src/common/reducers/modals/actions.js

```diff
@@ -39,3 +39,21 @@ export function closeAllModals() {
     }, 220);
   };
 }
+
+export function reopenModal(modalType, modalProps = {}) {
```

Comment:
What's this for?

Reply:
Uhm… could be a leftover from an Experiment 😅

Reply:
Ah no, that is a function who reopens an already opened modal whit new props. That’s made to get an animation when switching the os.